pages/index.tsx

```diff
@@ -37,7 +37,7 @@ export default function Home() {
 
         <HeroBanner {...content.heroBanner} aria-label={content.heroBanner.ariaLabels?.ctaButton} />
 
-        <SectionContainer>
+        <SectionContainer id="button-group">
```

Comment:
```suggestion
        <SectionContainer id="home-nav-section">
```
If we think that this `id` should be unique, and we use this kind of name (`button-group`), its not too much descriptive and can easily be duplicated in another component used inside this `home page`.

What do you think if we change to something more specific to this page? I gave one suggestion but feel free to use anything else.

Reply:
Thank you @LeoFuna for the improvement. So ids should be unique among all pages or being unique in one page is sufficient?

Reply:
@LeoFuna just at the same page is sufficient.